controllers/lvm_volumegroup.go

```diff
@@ -113,3 +136,14 @@ func (c lvmVG) getLvmVolumeGroups(r *LVMClusterReconciler, instance *lvmv1alpha1
 	}
 	return lvmVolumeGroups
 }
+
+func doesVGExistOnHosts(ctx context.Context, volumeGroup string, instance *lvmv1alpha1.LVMCluster) (bool, error) {
+
+	dcStatuses := instance.Status.DeviceClassStatuses
+	for _, dc := range dcStatuses {
+		if dc.Name == volumeGroup {
```

Comment:
- only for my understanding, we are finding whether VG exists or not by means of status of lvmcluster cr which inturn gets the info from groupnodestatus cr right?
- is it possible to club volumegroup and groupnodestatus crd? (if past discussions were a no, then nntr)

Reply:
> * only for my understanding, we are finding whether VG exists or not by means of status of lvmcluster cr which inturn gets the info from groupnodestatus cr right?
Correct
> * is it possible to club volumegroup and groupnodestatus crd? (if past discussions were a no, then nntr)
Can you elaborate?

Reply:
> Can you elaborate?
- not related to this PR, can be taken up separately
- in this single function either directly or indirectly we are using all the three CRs with some duplicate info, even though we discussed in length about the current state, if possible we want to re-look in reducing the duplicate info there by reducing num of CRs